modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowDefinitionImpl.java

```diff
@@ -66,6 +66,9 @@ public WorkflowDefinitionImpl() {
   @XmlElement(name = "description")
   private String description;
 
+  @XmlElement(name = "displayOrder")
+  private int displayOrder = 0;
```

Comment:
Just wondering if you wouldn't want a default display order of e.g. 100. That way it's easier to specify the one workflow to come first without having to touch all others.

Reply:
Thinking about this once again, I could set it to `-1` so `0` should be fine.